add_gem_package.sh

```diff
@@ -167,7 +158,7 @@ else
 	UPDATE=false
 fi
 
-if [[ -z $GEM_NAME ]] || [[ -z $TEMPLATE_NAME ]] || [[ $UPDATE != true ]] && [[ -z $REPO ]]; then
+if [[ -z $GEM_NAME ]] || [[ -z $TEMPLATE_NAME ]] || [[ $UPDATE != true ]]; then
```

Comment:
This breaks now when adding a new package because `UPDATE=false` and then `false != true` is always true.